src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.services.DataFileReader;
+import core.basesyntax.services.DataFileWriter;
+import core.basesyntax.services.DataToMapParser;
+import core.basesyntax.services.FruitShopService;
+import core.basesyntax.services.operations.Operable;
+import core.basesyntax.services.operations.PurchaseOperation;
+import core.basesyntax.services.operations.SupplyOperation;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MainApp {
+    private static final String FILE_PATH = "src/test/resources/inputFile.csv";
+    private static final String RESULT_FILE_PATH = "src/test/resources/resultFile.csv";
+    private static final String SUCCESSFUL_MESSAGE
+            = "file processed successfully, output file was created";
+    private static final String UNSUCCESSFUL_MESSAGE
+            = "file processed successfully, output file was created";
+    private static final String SUPPLY = "s";
+    private static final String BUY = "b";
+    private static final String RETURN = "r";
+
+    public static void main(String[] args) {
+        FruitShopService shopFileService = new FruitShopService(new DataFileReader(),
+                new DataFileWriter(), new DataToMapParser());
+        if (shopFileService.processFile(FILE_PATH,
+                RESULT_FILE_PATH, createOperationsMap())) {
+            System.out.println(SUCCESSFUL_MESSAGE);
```

Comment:
if you are using this message only once, you can use it directly here. It is not necessary to create a constant for it.

Reply:
Ok, constants was removed